ProcessLib/BoundaryCondition/NeumannBoundaryConditionLocalAssembler.h

```diff
@@ -69,6 +69,9 @@ class NeumannBoundaryConditionLocalAssembler final
 private:
     Parameter<double> const& _neumann_bc_parameter;
     typename Base::NodalVectorType _local_rhs;
+
+public:
+    EIGEN_MAKE_ALIGNED_OPERATOR_NEW;
```

Comment:
See this link https://eigen.tuxfamily.org/dox-devel/group__TopicStructHavingEigenMembers.html for details and it is defined in Core/util/Memory.h in Eigen.